src/core/DrumSynth.cpp

```diff
@@ -258,7 +250,9 @@ int DrumSynth::GetPrivateProfileString(
 	if (len == 0)
 	{
 		len = strlen(def);
-		strncpy(buffer, def, size);
+		const auto maxlen = std::min(len, size - 1);
+		std::memcpy(buffer, def, maxlen);
+		if (maxlen < len) { buffer[maxlen] = 0; }
```

Comment:
```suggestion
		buffer[maxlen] = '\0';
```
I could be wrong but it looks like it could be missing a null terminator if `maxlen >= len`.

Reply:
On line 253 I declare `maxlen = std::min(len, size - 1)`, so it is not possible for `maxlen` to be greater than `len`. However, `len` is likely to be equal to `maxlen` (it's only not if the string is huge for some reason), so you would be correct in that case. Thanks for catching that!